src/coverage.ts

```diff
@@ -49,13 +66,18 @@ export const coverage = async ({ token, ghToken, summary }: Props) => {
   const comparison = res?.check
   const utoken = res?.metadata?.token
 
+  let patchCoverage: number | undefined
```

Comment:
Move this into its own function that receives all the params it needs and returns the patch coverave  and some more meta information if needed.

Reply:
done